KenticoCloud.Delivery/Configuration/NET45ConfigurationProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.Configuration;
+
+#if (NET45)
+using System.Configuration;
+#endif
+
+namespace KenticoCloud.Delivery
+{
+    internal class NET45ConfigurationProvider : ConfigurationProvider, IConfigurationSource
```

Comment:
Let's call it `AppSettingsProvider` (or `ConfigurationManagerProvider`)...whichever you like better.